static/talawa/index.html

```diff
@@ -38,957 +38,57 @@
+      <h2 id="talawa-components">Talawa Components</h2>
+      <p><code>talawa</code> has these major software components:</p>
+      <ol>
+        <li><strong>talawa</strong>: <a href="https://github.com/PalisadoesFoundation/talawa">A mobile application with social media features</a></li>
+        <li><strong>talawa-api</strong>: <a href="https://github.com/PalisadoesFoundation/talawa-api">An API providing access to user data and features</a></li>
+        <li><strong>talawa-admin</strong>: <a href="https://github.com/PalisadoesFoundation/talawa-admin">A web based administrative portal</a></li>
+        <li><strong>talawa-docs</strong>: <a href="https://github.com/PalisadoesFoundation/talawa-docs">The online documentation website</a></li>
+      </ol>
+      <h2 id="documentation">Documentation</h2>
+      <ul>
+        <li>Read our <a href="https://github.com/PalisadoesFoundation/talawa/blob/develop/INSTALLATION.md">Talawa Mobile App developer installation guide</a> to get setup quickly.</li>
+        <li>The complete talawa documentation set can be found <a href="https://docs.talawa.io/talawa-docs/">here</a>.</li>
+        <li>Want to contribute? Look at <a href="https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md">CONTRIBUTING.md</a> to get started.</li>
+        <li>Visit the <a href="https://github.com/PalisadoesFoundation/talawa">Talawa GitHub</a> to see the code.</li>
+      </ul>
+    </section>
+  </div> 
 
   <div id="dartdoc-sidebar-left" class="sidebar sidebar-offcanvas-left">
     <header id="header-search-sidebar" class="hidden-l">
-  <form class="search-sidebar" role="search">
-    <input type="text" id="search-sidebar" autocomplete="off" disabled class="form-control typeahead" placeholder="Loading search...">
-  </form>
-</header>
-
-<ol class="breadcrumbs gt-separated dark hidden-l" id="sidebar-nav">
-  <li><a href="https://github.com/PalisadoesFoundation/talawa">talawa package</a></li>
-</ol>
+      <form class="search-sidebar" role="search">
+        <input type="text" id="search-sidebar" autocomplete="off" disabled class="form-control typeahead" placeholder="Loading search...">
+      </form>
+    </header>
 
+    <ol class="breadcrumbs gt-separated dark hidden-l" id="sidebar-nav">
+      <li><a href="https://github.com/PalisadoesFoundation/talawa">talawa package</a></li>
+    </ol>
 
-    <h5 class="hidden-xs"><span class="package-name">talawa</span> <span class="package-kind">package</span></h5>
```

Comment:
@Yass32 Could you tell me why this was deleted?

Reply:
> @Yass32 Could you tell me why this was deleted?

![Screenshot (34)](https://user-images.githubusercontent.com/108995581/224459645-850d9a53-1128-4554-879e-5076899f09bc.png)

I felt like there was no need to repeat 'talawa package' twice


And also I created an issue to fix the 'N|solid' image but didn't get a reply